### ISSUE TITLE
winreg error

### ISSUE THREAD

**Issue by graememilne** [ravenstx/PlaneNotifier#1]
![Screenshot_20230301_182755](https://user-images.githubusercontent.com/19634301/222233505-91237f6d-ea32-4e37-8009-298bfb27b281.png)

So thought this would be a cool project to try.
Followed the instructions (i have python 3.9 installed)
Started the program python3 planenotifier.py
I get this error,please see attached screen capture.
Any idea what the problem is ??
Thank you in advance

**Comment by ravenstx** [comment]
Yes currently it only works on windows 10 since i am using the python module 'winotify' to send notifications.
Feel free to contribute (have a look at this: https://pypi.org/project/py-notifier/).

**Comment by graememilne** [comment]
Ok i am confused now .
So i install this on linux computer
 *install the requirements*

cd PlaneNotifier
python -m pip install -r requirements.txt

YES/NO

**Comment by ravenstx** [comment]
No, it won't work on Linux for now.
To make it work i would need to use a different notification package and change the method print_notification in the file flight.py
